### PR TITLE
CI: Mark broken daily check with FIXME request

### PR DESCRIPTION
**Pull request by stappersg**
The daily CI action is broken for a long time. It is visible in our "upstream" git repository, a.k.a. the Canonical git repo, and all the forks of it. It emits "We don't care".

To prevent getting further used to "Yeah, the CI tests fail", modified the CI test into something that runs successfully.

## Proposed Commit Message

```
<type>(optional scope): <summary>  # no more than 72 characters

A description of what the change being made is and why it is being
made if the summary line is insufficient.  This should be wrapped at
72 characters.

If you need to write multiple paragraphs, feel free.

Fixes GH-NNNNN (GitHub Issue number. Remove line if irrelevant)
LP: #NNNNNN (Launchpad bug number. Remove line if irrelevant)
```

## Merge type

- [x] Squash merge using "Proposed Commit Message"
- [ ] Rebase and merge unique commits. Requires commit messages per-commit each referencing the pull request number (#<PR_NUM>)

### CODE REVIEW

**Comment by holmanb** [comment]
@stappersg this proposal doesn't actually solve anything. It should be a straightforward fix, just haven't had time yet.

**Comment by stappersg** [comment]
WARNING:  Incoming compliment


On Mon, Nov 18, 2024 at 12:30:29PM -0800, Brett Holman wrote:
> It should be a straightforward fix, just haven't had time yet.

Thank you for taking time to express that.



Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by stappersg** [comment]
> It should be a straightforward fix

[_Bluf_](#5929)
